Exiled.API/Features/FriendlyFireStatusCode.cs

```diff
@@ -0,0 +1,35 @@
+// -----------------------------------------------------------------------
+// <copyright file="FriendlyFireStatusCode.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    /// <summary>
+    /// Status code for adding FriendlyFire rules.
+    /// </summary>
+    public enum FriendlyFireStatusCode
+    {
+        /// <summary>
+        /// Adding specific role failed, unable to be added.
+        /// </summary>
+        UnableToAdd,
+
+        /// <summary>
+        /// Current role successfully added
+        /// </summary>
+        AddedRole,
+
+        /// <summary>
+        /// Partial roles were added, cannot guarantee all.
+        /// </summary>
+        AddedSomeRoles,
+
+        /// <summary>
+        /// All roles were successfully added
+        /// </summary>
+        AddedAllRoles,
```

Comment:
Not necessary - just use "Success" instead. Documentation can just be "All roles were successfully added"

Reply:
Done, thanks